lib/songs_row.dart

```diff
@@ -36,21 +45,20 @@ class SongRow extends StatelessWidget {
                     'Sing-Off',
                   ),
                 ),
-                Container(
-                  child: InkWell(
-                    onTap: () {
-                      // changeFeed(true);
-                      // changeSongs(false);
-                      //perhaps pick a random happy song
-                    },
-                    child: Icon(
-                      Icons.mood,
-                    ),
+                  Container(
+                    child: InkWell(
+                        onTap: () {
+                          changeSongs(false);
+                          changeRecorder(true);
+                          setSelectedSong(allSongs[0]);
```

Comment:
I guess this is the random song icon. Let's remember to change it to select an actual random song :)